src/NosCore.Parser/Parsers/NpcMonsterParser.cs

```diff
@@ -0,0 +1,495 @@
+                    hPbasup = 65;
+                }
+
+                if (i < 41)
+                {
+                    continue;
+                }
+
+                if ((99 - i) % 8 == 0)
+                {
+                    hPbasup++;
+                }
+            }
+
+            //Race == 0
+            _basicPrimaryMp[0] = 10;
+            _basicPrimaryMp[1] = 10;
+            _basicPrimaryMp[2] = 15;
+
+            var primaryBasup = 5;
+            byte count = 0;
+            var isStable = true;
+            var isDouble = false;
+
+            for (uint i = 3; i < 100; i++)
+            {
+                if (i % 10 == 1)
+                {
+                    _basicPrimaryMp[i] += _basicPrimaryMp[i - 1] + primaryBasup * 2;
+                    continue;
+                }
+
+                if (!isStable)
+                {
+                    primaryBasup++;
+                    count++;
+
+                    if (count == 2)
+                    {
+                        if (isDouble)
+                        {
+                            isDouble = false;
+                        }
+                        else
+                        {
+                            isStable = true;
+                            isDouble = true;
+                            count = 0;
+                        }
+                    }
+
+                    if (count == 4)
+                    {
+                        isStable = true;
+                        count = 0;
+                    }
+                }
+                else
+                {
+                    count++;
+                    if (count == 2)
+                    {
+                        isStable = false;
+                        count = 0;
+                    }
+                }
+
+                _basicPrimaryMp[i] = _basicPrimaryMp[i - (i % 10 == 2 ? 2 : 1)] + primaryBasup;
+            }
+
+            // Race == 2
+            _basicSecondaryMp[0] = 60;
+            _basicSecondaryMp[1] = 60;
+            _basicSecondaryMp[2] = 78;
+
+            var secondaryBasup = 18;
+            var boostup = false;
+
+            for (uint i = 3; i < 100; i++)
+            {
+                if (i % 10 == 1)
+                {
+                    _basicSecondaryMp[i] += _basicSecondaryMp[i - 1] + (int)i + 10;
+                    continue;
+                }
+
+                boostup = !boostup;
+                secondaryBasup += boostup ? 3 : 1;
+                _basicSecondaryMp[i] = _basicSecondaryMp[i - (i % 10 == 2 ? 2 : 1)] + secondaryBasup;
+            }
+
+            // basicXPLoad
+            for (var i = 0; i < 100; i++)
+            {
+                _basicXp[i] = i * 180;
+            }
+
+            // basicJXpLoad
+            for (var i = 0; i < 100; i++)
+            {
+                _basicJXp[i] = 360;
+            }
+
+        }
+
+        private short ImportHeroXp(Dictionary<string, string[][]> chunk)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ImportHeroXp'.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4512441)